### PR TITLE
Helper for adding hooks in upgrade scripts

### PR DESCRIPTION
**Pull request by kpodemski**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | This helper makes adding hooks even easier and eases the process of review of such PRs
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | n/a
| Sponsor company   | Mr Podemsky Corp
| How to test?      | Install 8.0.1, use helper in 8.0.2 upgrade script and upgrade, and see if a new hook exists. Developer feature, developer QA probably.

How to use it?

`/* PHP:add_hook('mySuperAwesomeHook', 'Super awesome hook', 'This is yet another awesome hook in PrestaShop'); */;`

### CODE REVIEW

**Review by Hlavtox — APPROVED** [review]
Hlavacek & Son approves

File: upgrade/php/add_hook.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+function add_hook($hook, $title, $description, $position = 1)
```
Comment (zuk3975) [inline upgrade/php/add_hook.php:26]: could you type the arguments?

**Comment by zuk3975** [comment]
testing

**Comment by zuk3975** [comment]
QA :heavy_check_mark: 
Screenshot from 2023-04-14 17-15-32

**Pull request merged**